core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -417,7 +417,10 @@ object Battle {
     private fun addXp(thisCombatant: ICombatant, amount: Int, otherCombatant: ICombatant) {
         var baseXP = amount
         if (thisCombatant !is MapUnitCombatant) return
-        if (thisCombatant.unit.promotions.totalXpProduced() >= thisCombatant.unit.civInfo.gameInfo.ruleSet.modOptions.maxXPfromBarbarians
+        if (
+            (thisCombatant.unit.promotions.totalXpProduced() >= thisCombatant.unit.civInfo.gameInfo.ruleSet.modOptions.maxXPfromBarbarians
+                || thisCombatant.unit.promotions.totalXpProduced() >= thisCombatant.unit.civInfo.gameInfo.ruleSet.modOptions.constants.maxXPfromBarbarians
```

Comment:
Equivalent to `totalXpProduced() >= min(modOptions.maxXPfromBarbarians, modOptions.constants.maxXPfromBarbarians)`? I think mods would have to define both properties to be able to increase the threshold.

Reply:
The `modOptions.maxXPfromBarabrians` is only temporary and should be removed once most mods have updated, so upping both should be unnecessary.
It should be a `min` if you assume mods will lower the amount, and a `max` otherwise, so I'll work something out for that before merging

Reply:
Oh, I meant the current behaviour is equivalent to a min, which probably isn't intended as it prevents the mod from being able to increase the value unless it increases both properties.

Maybe checking only the new property, and setting it from the old one on init if the old one's changed and the new one's unchanged?